packages/model-runtime/src/providers/replicate/index.ts

```diff
@@ -0,0 +1,619 @@
+        }
+      }
+      if (seed !== undefined && seed !== null) {
+        input.seed = seed;
+        this.debugLog('[Replicate createImage] Set seed:', seed);
+      }
+
+      // Run prediction - with useFileOutput: false, returns URL strings
+      // Log input object without Buffer data (which would be huge)
+      const inputForLogging = { ...input };
+      for (const key in inputForLogging) {
+        if (Buffer.isBuffer(inputForLogging[key])) {
+          inputForLogging[key] = `<Buffer ${inputForLogging[key].length} bytes>`;
+        }
+      }
+      this.debugLog(
+        '[Replicate createImage] Final input object:',
+        JSON.stringify(inputForLogging, null, 2),
+      );
+      this.debugLog('[Replicate createImage] Calling client.run...');
+
+      const output = await this.client.run(model as any, { input });
+
+      this.debugLog('[Replicate createImage] Raw output:', output);
+      this.debugLog(
+        '[Replicate createImage] Output type:',
+        typeof output,
+        'Is array:',
+        Array.isArray(output),
+      );
+
+      // Extract URL from output
+      let outputImageUrl: string;
+
+      if (Array.isArray(output)) {
+        if (output.length === 0) {
+          throw new Error('Replicate returned empty array');
+        }
+        // First item should be the URL string
+        outputImageUrl = output[0];
+        this.debugLog('[Replicate] Extracted URL from array:', outputImageUrl);
+      } else if (typeof output === 'string') {
+        outputImageUrl = output;
+        this.debugLog('[Replicate] Output is direct string URL:', outputImageUrl);
+      } else {
+        // eslint-disable-next-line no-console
+        console.error('[Replicate] Unexpected output structure:', output);
+        throw new Error(`Unexpected output format from Replicate: ${typeof output}`);
+      }
+
+      if (typeof outputImageUrl !== 'string') {
+        throw new Error(`Expected URL string, got ${typeof outputImageUrl}: ${outputImageUrl}`);
+      }
+
+      this.debugLog('[Replicate] Final imageUrl:', outputImageUrl);
+
+      return {
+        height: height,
+        imageUrl: outputImageUrl,
+        width: width,
+      };
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('[Replicate createImage] ERROR caught:', error);
+      // eslint-disable-next-line no-console
+      console.error('[Replicate createImage] Error type:', (error as any)?.constructor?.name);
+      // eslint-disable-next-line no-console
+      console.error('[Replicate createImage] Error message:', (error as any)?.message);
+      // eslint-disable-next-line no-console
+      console.error('[Replicate createImage] Full error:', JSON.stringify(error, null, 2));
+      throw this.handleError(error);
+    }
+  }
+
+  /**
+   * Image generation support (FLUX, Stable Diffusion, etc.)
+   */
+  async textToImage(payload: TextToImagePayload): Promise<string[]> {
```

Comment:
We will remove the textToImage method in the future.

Reply:
It was originally designed for the dalle plugin, and the dalle plugin will be deprecated later. The image page will use the createImage method to handle image generation. You only need to implement createImage.